### DISCUSSION TITLE
Power of Persuasion

### DISCUSSION THREAD

**Discussion by FrozenEmpire** [ChilliHugger/The-Lords-Of-Midnight#185] [Code Algorithm Questions] [answered]
Hi Chris,
Tonight's query...

Objects have a flag  of_recruitment which is checked during recruitment via the property CanHelpRecruitment()

if ( this->Carrying()!= NULL && this->Carrying()->CanHelpRecruitment() )
      success+=2;

Checking in the exports that I made from TME data at the beginning of DDR development, it would seem that no objects have this flag set in my exports.

I was expecting perhaps the Crowns (that have Powers of Persuasion) to have this flag set.

Can you please check for a potential bug in the TME data. Thanks

(Note, this flag does not appear in the tblObjects.tsv file either ?)

**Comment by IcemarkUK** [comment, marked as answer]
Good catch. I'm not sure where this regressed. I've got a feeling I updated the data during the first release 2013, and then somehow not used that particular binary data. Or I added code in to the Scenario IntialiseAfterCreate to set the flags on the objects because I was working around not having MSAccess at that time and updating stuff in code, and later deleted it. Or, and more likely, I just never updated the objects!

The actual original check was just seeing if the character carried objects 0 - 4

- OB_CROWN_VARENAND
- OB_CROWN_CARUDRIUM
- OB_SPELL_THIGRORN
- OB_RUNES_FINORN
- OB_CROWN_IMIRIEL

```
L7D2F: MOV    BYTE PTR CharacterNumber,AL
       CALL   GetCharObject                                    ;
       CMP    AL,004h                                          ;Check to see if he
       JZ     L7D3B                                            ;has any crowns of
       JNC    L7D3D                                            ;Persuassion.
L7D3B: INC    CL                                               ;Helps Alot if he
       INC    CL                                               ;does.
```

**Reply by FrozenEmpire** [reply]
Cool. Thanks for checking.
